src/bionics.h

```diff
@@ -74,6 +74,10 @@ struct bionic_data {
     * If true, this bionic is included with another.
     */
     bool included = false;
+    /**Factor modifiying weight capacity*/
+    float weight_capacity_modifier;
```

Comment:
This should be `units::weight` and one should be able to use weight strings (like "10 kg") in JSON.